src/clients/bridges/AdapterManager.ts

```diff
@@ -71,51 +90,71 @@ export class AdapterManager {
       this.adapters[LINEA] = new LineaAdapter(logger, spokePoolClients, filterMonitoredAddresses(LINEA));
     }
     if (this.spokePoolClients[MODE] !== undefined) {
-      this.adapters[MODE] = new OpStackAdapter(
+      const bridges = constructBridges(MODE);
+      this.adapters[MODE] = new BaseChainAdapter(
+        spokePoolClients,
         MODE,
+        hubChainId,
+        filterMonitoredAddresses(MODE),
         logger,
         SUPPORTED_TOKENS[MODE],
-        spokePoolClients,
-        filterMonitoredAddresses(MODE)
+        bridges,
```

Comment:
Is `bridges` used anywhere else? Could maybe squash it?

```suggestion
        constructBridges(MODE),
```

Reply:
Nice. Yeah we can https://github.com/across-protocol/relayer/pull/1824/commits/0d9e84658ad4d8385522e9d91bfaf37549b20fb4